compiler/rustc_resolve/src/effective_visibilities.rs

```diff
@@ -11,6 +11,7 @@ use rustc_middle::middle::privacy::Level;
 use rustc_middle::middle::privacy::{EffectiveVisibilities, EffectiveVisibility};
 use rustc_middle::ty::Visibility;
 use std::mem;
+use tracing::info;
```

Comment:
Are all of these additions from tracing in different files needed?

Reply:
Yes. the `#[macro_use] extern crate tracing;` gave us implicit import of all `tracing` macros. With that gone, we must do explicit imports, and `tracing` macros are used in lots of places.